### PR TITLE
Remove conan-center default remote

### PR DESCRIPTION
**Pull request by danimtb**
Docs for https://github.com/conan-io/conan/pull/9401

### CODE REVIEW

File: uploading_packages/remotes.rst [hunk 1]
```diff
@@ -55,21 +55,22 @@ To contribute packages to ConanCenter, read the :ref:`ConanCenter guide <conan_c
 conan-center [deprecated]
 -------------------------
 
-**conan-center** was the official repository but is is no longer recommended. It is configured as the second default remote in the Conan
-client to keep backwards compatibility:
+**conan-center** was the official repository but is is **no longer a default remote** in the Conan client and **its usage is completely
```
Comment (danimtb) [inline uploading_packages/remotes.rst:58]: Left this section for reference purposes (as some people might still be using the remote until we completely shut it down), but no objections if we want to remove it, as this is documented in previous versions of the documentation (although maybe not so easy to find it).

File: uploading_packages/remotes.rst [hunk 2]
```diff
@@ -55,21 +55,22 @@ To contribute packages to ConanCenter, read the :ref:`ConanCenter guide <conan_c
...
+discouraged**. This documentation is kept here only for reference purposes.
 
 .. code-block:: bash
 
     $ conan-center: https://conan.bintray.com [Verify SSL: True]
 
-It contains all the packages from the ConanCenter remote as well as **legacy packages with full reference** (`zlib/1.2.11@conan/stable`).
-These package binaries were created by users in their own Bintray repositories and included in this main repository. This flow of
-contributing packages to ConanCenter is no longer available and packages are **not recommended** and should be considered as **legacy**.
+It contains all the packages that were in the `conancenter` remote until July 1st (new packages are no longer uploaded to this remote),
```
Comment (jgsogo) [inline uploading_packages/remotes.rst:65]: This `conancenter` is misleading, as it is the name of the current remote

```suggestion
It contains all the packages that were uploaded to ConanCenter before July 1st (new packages are no longer uploaded to this remote),
```

File: uploading_packages/remotes.rst (hunk 1 above)
Comment (jgsogo) [inline uploading_packages/remotes.rst:58]:
```suggestion
**conan-center** was the official repository but is **no longer a default remote** in the Conan client and **its usage is completely
```

**Review by lasote — APPROVED** [review]
Accept the @jgsogo suggestions and it is well done!

**Review by czoido — APPROVED** [review]

**Pull request merged**